uk.co.compucorp.civicrm.hrcontactactionsmenu/js/contactactions.js

```diff
@@ -1,11 +1,24 @@
-(function ($) {
+(function ($, CRM) {
```

Comment:
Please make sure you are adding backstop tests for new items, and also old tests are passing.